generator/src/main/java/com/scottlogic/deg/generator/guice/BaseModule.java

```diff
@@ -92,6 +94,9 @@ protected void configure() {
 
         bind(VelocityMonitor.class).in(Singleton.class);
         bind(JavaUtilRandomNumberGenerator.class).toInstance(new JavaUtilRandomNumberGenerator(LocalDateTime.now().getNano()));
+
+        //bind(ProfileSchemaValidator.class).to(ProfileSchemaValidatorMedeia.class);
```

Comment:
It would be best to include a comment with this comment(!) to explain why its commented out and so people don't remove it as part of a refactor